src/debug.c

```diff
@@ -1049,6 +1049,14 @@ __attribute__((noinline, weak)) void _serverAssert(const char *estr, const char
     bugReportEnd(0, 0);
 }
 
+/* Returns the argv argument in binary representation, limited to length 128. */
+sds getArgvReprString(robj *argv) {
+    robj *decoded = getDecodedObject(argv);
```

Comment:
getDecodedObject can allocate memory. We used a buffer on the stack for this before (`char buf[128]`). I think the reason is that we want to avoid memory allocation in the crash handler code, becuase the crash can be because of OOM?

Reply:
It is a good point. In this case  it is a assert crash report. It doesn't seem to have the memory issue. 

And in the other one, it can cause by a oom and it is already doing the alloca though

Reply:
line 1900

Reply:
Ah, it was already using getDecodedObject. OK, then, approved. :)